crypto/evp/m_sigver.c

```diff
@@ -301,6 +302,12 @@ int EVP_DigestSignUpdate(EVP_MD_CTX *ctx, const void *data, size_t dsize)
                                                       data, dsize);
 
  legacy:
+    /* do_sigver_init() checked that |digest_custom| is non-NULL */
+    if (pctx->flag_call_digest_custom
+        && !ctx->pctx->pmeth->digest_custom(ctx->pctx, ctx))
```

Comment:
Actually the reason why we put this part in the init phase back in 2018 was to avoid multi-part code copy - but if that is not possible anymore, then this seems fine as well.

Reply:
Well, one of the reasons...

Reply:
I think this is justifiable by making it more manageable for the user. It's not like we're copying *large* chunks of code, this is, after all, just two lines (plus a comment)